ccdproc/core.py

```diff
@@ -1740,6 +1741,94 @@ def _sigma_mask(baseline, one_sigma_value, lower_sigma, upper_sigma):
     return mask
 
 
+def bitmask2mask(bitmask, ignore_bits=0, flip_bits=None):
```

Comment:
@crawfordsm Yes, in _my_ opinion (after spending significant time "educating" myself on this), ``bitfields_to_boolean_mask`` seems better than the old name that I have used ``bitmask2mask`` (which was technically incorrect IMO), or the other proposed name ``apply_bitmask`` which suggests the operation of "applying" (e.g., ``logical-AND``) a mask: ``dq&mask``.

However, it is exhausting to discuss semantics and moreover it is your project so please feel free to name things in any way that you feel is more intuitive (and/or correct). In my above comment about masks and fields I just wanted to insert what I have gathered from around the web.